aviasales/src/app/app.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="app">
+  <img src="Logo.svg" alt="logo" class="logo">
+
+  <div class="container">
+    <app-filter
+      (onFilterChange)="ticketsParams.filters = $event; getShownTickets()"
```

Comment:
Don't understand this part **"ticketsParams.filters = $event; getShownTickets()"**